js/templates/modals/listingDetail/listing.html

```diff
@@ -2,8 +2,19 @@
 
 <div class="modalContentInner">
   <div class="listingControls flex">
+    <div class="jsModalClose contentBox clrP clrBr clrSh3" style="height: 35px; line-height: 35px; padding: 0 8px; cursor: pointer">
+      <div class="userIcon floL discTn clrBr2 clrSh1"
+        <% var avatarHash = ob.avatarHashes ? ob.isHiRez() ? ob.avatarHashes.small : ob.avatarHashes.tiny : ''; %>
+        <% if (avatarHash) { %>
+          style="background-image: url(<%= ob.getServerUrl(`ipfs/${avatarHash}`) %>), url('../imgs/defaultAvatar.png')"
+        <% } else { %>
+         style="background-image: url('../imgs/defaultAvatar.png');"
+        <% } %>>
+      </div>
+      <div style="overflow: auto; font-size: 13px">Store Name <u style="font-size: 12px">View store</u></div>
```

Comment:
It's on my radar to wire this in as much as possible.

On the store we have the data. On search (at least the 3rd party search), we'll also have the data. Not sure about channel, but I think we'll have the data there too since the little listing card needs some basic profile data.

The only case I'm seeing where we might not have this data is if you directly enter a channel based listing url (e.g. @themes/channel/big-brown-shoe) in the address bar, so we'd probably just need to lazy load it in there. Depending on how we design the channel, it's possible we may be lazy loading many things related to the underlying channel page, so that if and when the user closes the modal, they have some semblance of a channel waiting for them.